node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -133,56 +133,12 @@ class NodeRuntime private[node] (
                           .toEffect
 
       _ <- Task.delay {
-            val influxdb = conf.kamon.influxDb
-              .map { i =>
-                val authentication = i.authentication
-                  .map { a =>
-                    s"""
-                    |    authentication {
-                    |      user = "${a.user}"
-                    |      password = "${a.password}"
-                    |    }
-                    |""".stripMargin
-                  }
-                  .getOrElse("")
-
-                s"""
-                |  influxdb {
-                |    hostname = "${i.hostname}"
-                |    port = ${i.port}
-                |    database = "${i.database}"
-                |    protocol = "${i.protocol}"
-                |    $authentication
-                |  }
-                |""".stripMargin
-              }
-              .getOrElse("")
-            val kamonConf =
-              s"""
-               |kamon {
-               |  environment {
-               |    service = "rnode"
-               |    instance = "${id.toString}"
-               |  }
-               |  metric {
-               |    tick-interval = 10 seconds
-               |  }
-               |  system-metrics {
-               |    host {
-               |      enabled = ${conf.kamon.sigar}
-               |      sigar-native-folder = ${conf.server.dataDir.resolve("native")}
-               |    }
-               |  }
-               |  $influxdb
-               |}
-               |""".stripMargin
-            Kamon.reconfigure(ConfigFactory.parseString(kamonConf).withFallback(Kamon.config()))
-            if (conf.kamon.influxDb.isDefined)
-              Kamon.addReporter(new kamon.influxdb.InfluxDBReporter())
+            Kamon.reconfigure(conf.underlying.withFallback(Kamon.config()))
+            if (conf.kamon.influxDb) Kamon.addReporter(new kamon.influxdb.InfluxDBReporter())
             if (conf.kamon.prometheus) Kamon.addReporter(prometheusReporter)
             if (conf.kamon.zipkin) Kamon.addReporter(new ZipkinReporter())
             Kamon.addReporter(new JmxReporter())
-            SystemMetrics.startCollecting()
+            if (conf.kamon.sigar) SystemMetrics.startCollecting()
```

Comment:
cigar? sugar?

Reply:
Kamon System Metrics uses [Sigar System Information](https://github.com/hyperic/sigar) under the hood. This option enables them. I had to add this flag because of an issue with JRE 10+. This issue has been resolved. So this flag will be removed in near future.